### PR TITLE
Update everything

### LINKED ISSUE #2: move tests to node-tests folder
migrated issue from the checklist in https://github.com/ember-cli/ember-cli-blueprint-test-helpers/issues/4

### LINKED ISSUE #3: Update Blueprints and Tests
Import from:
- https://github.com/emberjs/ember.js/pull/13029
- https://github.com/emberjs/data/pull/3813

see also https://github.com/ember-cli/ember-cli-legacy-blueprints/compare/master...trabus:blueprints

### PR DESCRIPTION
**Pull request by Turbo87**
see https://github.com/emberjs/ember.js/pull/13029

resolves #3 
resolves #2

### CODE REVIEW

**Comment by trabus** [comment]
Thank you for cleaning this up. Looking good.

**Comment by trabus** [comment]
@Turbo87 the travis failures may be related to https://github.com/ember-cli/ember-cli-blueprint-test-helpers/issues/33

**Comment by Turbo87** [comment]
haven't had time to investigate yet, but somethings wrong. it works on the emberjs PR, but I must've missed something here.

**Comment by Turbo87** [comment]
@trabus it seems that indeed I have the same issue as in https://github.com/ember-cli/ember-cli-blueprint-test-helpers/issues/33

**Comment by Turbo87** [comment]
@rwjblue any idea how to avoid setting `EMBER_DATA_SKIP_VERSION_CHECKING_DO_NOT_USE_THIS_ENV_VARIABLE=true`?

**Comment by trabus** [comment]
@Turbo87 I would recommend that you remove ember-data from this lib, and add the blueprints directly. The intent of this addon was to provide a snapshot of the blueprints in ember-cli at the state where we diverged, providing backward-compatibility support for consumers that are not using addonified ember and ember-data versions.

**Comment by Turbo87** [comment]
I tried removing the ember-data dependency but somehow Travis still failed with the same problem

**Comment by Turbo87** [comment]
@stefanpenner @rwjblue @trabus I think this is ready for review and merging now

File: blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import { initialize } from '<%= dasherizedModulePrefix %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+import destroyApp from '../../helpers/destroy-app';
```
Comment (stefanpenner) [inline blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js]: @trabus should these be absolute?

**Comment by stefanpenner** [comment]
this looks good, i left one question. Then we can merge

File: blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js (hunk 1 above)
Comment (Turbo87) [inline blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js]: note that those are the exact same files that are now merged into Ember.js too

see https://github.com/emberjs/ember.js/blob/master/blueprints/instance-initializer-test/qunit-files/tests/unit/instance-initializers/__name__-test.js#L4

**Pull request merged**